config/laravels.php

```diff
@@ -1,95 +1,465 @@
+    |
+    */
+
+    'processes' => [],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Timer
+    |--------------------------------------------------------------------------
+    |
+    | Wrapper cron job base on Swoole's Millisecond Timer, replace Linux Crontab.
+    |
+    | Read more at this link:
+    | https://github.com/hhxsv5/laravel-s/blob/master/README.md#millisecond-cron-job
+    |
+    */
+
+    'timer' => [
+        'enable' => env('LARAVELS_TIMER', false),
+
+        // The list of cron job
+        'jobs' => [],
+
+        // Max waiting time of reloading
+        'max_wait_time' => 5,
+
+        // Enable the global lock to ensure that only one instance starts the timer 
+        // when deploying multiple instances. 
+        // This feature depends on Redis https://laravel.com/docs/8.x/redis
+        'global_lock' => false,
         'global_lock_key' => config('app.name', 'Laravel'),
     ],
-    'swoole_tables'            => [],
-    'register_providers'       => [],
-    'cleaners'                 => [
-        // See LaravelS's built-in cleaners: https://github.com/hhxsv5/laravel-s/blob/master/Settings.md#cleaners
+
+    /*
+    |--------------------------------------------------------------------------
+    | Tables
+    |--------------------------------------------------------------------------
+    |
+    | All defined tables will be created before Swoole starting.
+    |
+    | Read more at this link:
+    | https://github.com/hhxsv5/laravel-s/blob/master/README.md#use-swooletable
+    |
+    */
+
+    'swoole_tables' => [],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Register Providers
+    |--------------------------------------------------------------------------
+    |
+    | The Service Provider list, will be re-registered each request, and run method boot() 
+    | if it exists. Usually, be used to clear the Service Provider 
+    | which registers Singleton instances.
+    |
+    | Read more at this link:
+    | https://github.com/hhxsv5/laravel-s/blob/master/Settings.md#register_providers
+    |
+    */
+
+    'register_providers' => [],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Cleaners
+    |--------------------------------------------------------------------------
+    |
+    | The list of cleaners for each request is used to clean up some residual 
+    | global variables, singleton objects, and static properties to avoid 
+    | data pollution between requests.
+    |
+    | Read more at this link:
+    | https://github.com/hhxsv5/laravel-s/blob/master/Settings.md#cleaners
+    |
+    */
+
+    'cleaners' => [
+        Hhxsv5\LaravelS\Illuminate\Cleaners\RequestCleaner::class,
+        Hhxsv5\LaravelS\Illuminate\Cleaners\AuthCleaner::class,
+        Hhxsv5\LaravelS\Illuminate\Cleaners\SessionCleaner::class,
+        App\Cleaners\SanctumCleaner::class,
```

Comment:
These cleaners should be removed.
`RequestCleaner` is built-in and enabled. `AuthCleaner` `SessionCleaner`  `SanctumCleaner` shouldn't be enabled default.